tools/config.ts

```diff
@@ -88,11 +88,11 @@ export const PROD_DEPENDENCIES = PROD_NPM_DEPENDENCIES.concat(APP_ASSETS);
 const SYSTEM_CONFIG_DEV = {
   defaultJSExtensions: true,
   paths: {
-    'main': `${APP_ROOT}main`,
```

Comment:
Oh, I was searching for it as I didn't remember the syntax. Knew I saw something like this somewhere. Would definitely help with readability.

I'll send an update later, thx :+1: